Automattic/simplenote-electron
# Updating the ui for the editing tags section

### Fix

As part of the UI Spring cleaning we are doing this looks to take care of updating the Edit Tags section. Resolves #2567.

This changes from using a cross icon on the left side of the tag name to using a trash icon on the right side. 
While editing the styles I also removed a number of lines that don't appear to be in use anymore at all. 

I also updated some of the styles to better use the classes in the themes.scss file instead of colors in the component styles. Not all are removed more refactoring will be needed. 

Removed two icons no longer needed `cross-outline-small` and `cross-outline`.

### Test

1. Go to menu to see tags
2. Tap on edit link
3. Notice the blue icon on the right side
4. Try in both light and dark mode theme

### Screenshots

<img width="269" alt="Screen Shot 2021-01-22 at 3 01 53 PM" src="https://user-images.githubusercontent.com/1326294/105533628-c85ed100-5cc2-11eb-8b23-15b51e06e0f1.png">

<img width="275" alt="Screen Shot 2021-01-22 at 2 58 51 PM" src="https://user-images.githubusercontent.com/1326294/105533411-7b7afa80-5cc2-11eb-93de-e58940371fe8.png">

### Release

- Update styles for edit tags panel including changing the cross outline icon to the trash icon.
